docker/runner.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ghcr.io/actions/actions-runner:2.316.1
```

Comment:
Here are a few docs for building alternative images based on older OS:

https://baccini-al.medium.com/how-to-containerize-a-github-actions-self-hosted-runner-5994cc08b9fb

https://gist.github.com/deptno/20c999efd4e00ee5e5fb9861db4a8ee1

I haven't tried any of them, so take it with a grain of salt.

Reply:
It doesn't have to have actions-runner in it. I took it because i thought it had most of the things we need. We can really base everything from ubuntu:20.04